### DISCUSSION TITLE
Failed to set cursor, split mode

### DISCUSSION THREAD

**Discussion by xdanep** [Isrothy/neominimap.nvim#145] [Q&A]
I have this issue, and this is my config.
![image](https://github.com/user-attachments/assets/77530c6b-fe10-43ce-9c9d-faf794ec0dc2)
![image](https://github.com/user-attachments/assets/4bf0b7f7-8734-4a52-8f86-ca3d508f865f)

**Comment by Isrothy** [comment]
Thank you for bringing this issue to our attention. To help me investigate and resolve the bug more efficiently, could you please provide the exact steps or procedures to reproduce the issue?

**Reply by xdanep** [reply]
Right after opening any file with neovim. I also noticed that the `sync_cursor=false` option does not work as there is no change from when it is set to true.

**Reply by Isrothy** [reply]
Could you provide a minimal neovim setup that can produce this issue?

**Reply by xdanep** [reply]
This is my current config
```lua
---@module "neominimap.config.meta"
return {
  "Isrothy/neominimap.nvim",
  version = "v3.*.*",
  enabled = true,
  lazy = false, -- NOTE: NO NEED to Lazy load
  -- Optional
  keys = {
      { "<leader>nt", "<cmd>Neominimap toggle<cr>", desc = "Toggle minimap" },
  },
  init = function()
    vim.opt.wrap = false -- Recommended
    vim.opt.sidescrolloff = 36 -- It's recommended to set a large value
    ---@type Neominimap.UserConfig
    vim.g.neominimap = {
      auto_enable = true,
      layout = "split", ---@type Neominimap.Config.LayoutType
      split = {
        close_if_last_window = true, ---@type boolean
      },
      sync_cursor = false,
    }
  end,
}
```

**Reply by Isrothy** [reply]
Do you have any other plugins installed?

**Reply by xdanep** [reply]
These are my plugins
![Screenshot_20240824_110237_GitHub.jpg](https://github.com/user-attachments/assets/0c1e3c4b-17ad-40ae-9cbb-22024fc06a7f)

**Reply by Isrothy** [reply]
Can you provide a configuration like this?

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
  spec = {
    -- add plugins here
  },
})
```
Save as repro.lua and run with `nvim -u repro.lua`
